frontend/jest.config.ts

```diff
@@ -35,6 +35,14 @@ const config: Config.InitialOptions = {
 			browsers: ['chromium', 'firefox', 'webkit'],
 		},
 	},
+	coverageThreshold: {
+		global: {
+			statements: 65,
+			branches: 50,
```

Comment:
We should have higher limits for newly added files and lower limit for existing ones